### PR TITLE
Jump with animation

### PR DESCRIPTION
**Pull request by DKevi**
Implement jump with animation;
Create a script for animation controller;

### CODE REVIEW

File: Assets/Scenes/SampleScene.unity
Comment (jialuogao) [inline Assets/Scenes/SampleScene.unity:309]: is this intended to have a 0 value? If yes just wondering how this value gets assigned?

File: Assets/Blink/Art/Characters/LowPoly/Demo_FREE_LowPolyCharacter/LowPolyHumanAnimator.controller
Comment (jialuogao) [inline Assets/Blink/Art/Characters/LowPoly/Demo_FREE_LowPolyCharacter/LowPolyHumanAnimator.controller:1]: Do we want to remove all these? If not we might want to add it back?

**Review by jialuogao — CHANGES_REQUESTED** [review]
LGTM, commented on some minor questions.
Also wonder where did those hooks file go? 
For LFS could you add that to .gitignore?
Thanks

**Comment by zyd2001** [comment]
> LGTM, commented on some minor questions. Also wonder where did those hooks file go? For LFS could you add that to .gitignore? Thanks

Those files should not be there if LFS is configured correctly, so it is no need to modify .gitignore

File: Assets/Scenes/SampleScene.unity
Comment (DKevi) [reply Assets/Scenes/SampleScene.unity:309]: They can be set to any value. The higher the jump power the higher the character will jump. 0 means no jump at all. It should have a field somewhere you can change easily while using the UI.

**Comment by AlstonXiao** [comment]
LGTM. Remember these unity description file (like scene.unity) is not really human readable. Make sure you resolve them in local before push and pull request. (1, merge dev to your branch. 2, resolve conflict. 3, push and pull request)

File: Assets/Scenes/SampleScene.unity
Comment (jialuogao) [reply Assets/Scenes/SampleScene.unity:309]: Ok, just wondering if we want a default value but not needed to modify in this PR

**Comment by jialuogao** [comment]
Ok, will unblock and approve, @DKevi do what @AlstonXiao asked, and after, feel free to merge if no additional changes are needed

**Review by jialuogao — DISMISSED** [review]

**Comment by zyd2001** [comment]
我试了一下，好像跳的过程中按空格也会触发下一次跳跃，应该算是bug吧

**Comment by jialuogao** [comment]
> 我试了一下，好像跳的过程中按空格也会触发下一次跳跃，应该算是bug吧

这不是isgrounded的检测吗？难道有bug，会不会因为跳的时候坐标没动，所以检测还在地上？ @DKevi 看一下吧

**Comment by johnw08** [comment]
> 我试了一下，好像跳的过程中按空格也会触发下一次跳跃，应该算是bug吧

这会不会是他设计的多段跳

**Comment by DKevi** [comment]
我这看貌似没啥问题...? @zyd2001 你设的参数多少，我复现一下
@johnw08 并没有多段跳呢还

**Comment by zyd2001** [comment]
> 我这看貌似没啥问题...? @zyd2001 你设的参数多少，我复现一下 @johnw08 并没有多段跳呢还

我就是直接checkout到你的branch运行的，我试试看弄个新的clone

**Comment by DKevi** [comment]
> > 我这看貌似没啥问题...? @zyd2001 你设的参数多少，我复现一下 @johnw08 并没有多段跳呢还
> 
> 我就是直接checkout到你的branch运行的，我试试看弄个新的clone

我和诺神研究了一下，是跳的太低了动画没结束但是人其实已经落地了，就会有这样的效果

**Review by jialuogao — APPROVED** [review]

**Pull request merged**
